client/src/Router.re

```diff
@@ -51,7 +51,7 @@ module Unload = {
         );
         None;
       });
-      React.null;
+      ReactOld.null;
```

Comment:
I don't understand this change :/ why not using `React.null`? One less thing to change later on.

Reply:
Remove `ReactOld`